sources/service-dns-cloudflare/main.tf

```
@@ -22,10 +22,15 @@ resource "cloudflare_record" "active_domain" {
 }
 
 resource "cloudflare_record" "passive_domain" {
-  for_each = local.passive_domains
+  for_each = {for y in local.passive_domains : y.name => {
+      host_tmp = split(".", y.name)
```

Can we rename this to make it easier to follow the intended logic flow?
```suggestion
      name_pieces = split(".", y.name)
```